amd_openvx/openvx/api/vx_api.cpp

```diff
@@ -944,13 +944,21 @@ VX_API_ENTRY vx_image VX_API_CALL vxCreateImageFromHandle(vx_context context, vx
                         data->children[i]->buffer = (vx_uint8 *)(ptrs ? ptrs[i] : nullptr);
                         data->children[i]->u.img.stride_in_bytes = addrs[i].stride_y;
                         data->children[i]->gpu_buffer_offset = 0;
+#if (ENABLE_OPENCL || ENABLE_HIP)
```

Comment:
Not sure why this is needed when creating the data

Reply:
@rrawther  for the createImageFromChannel test case I had to add this

Reply:
This is because of vxMapImagePatch which is copying from GPU to CPU. So input needs to be synced as well in the current implementation. Need to investigate this in future how to avoid this